src/controller/python/OpCredsBinding.cpp

```diff
@@ -700,4 +710,28 @@ PyChipError pychip_GetCompletionError()
     return ToPyChipError(sTestCommissioner.GetCompletionError());
 }
 
+PyChipError pychip_DeviceController_SetDACRevocationSetPath(const char * dacRevocationSetPath)
+{
+    chip::Credentials::DeviceAttestationRevocationDelegate * dacRevocationDelegate =
+        GetTestAttestationRevocationDelegate(dacRevocationSetPath);
```

Comment:
Doesn't this mean that you can only ever `dacRevocationSetPath` to be from the test attestation?

Reply:
Yes, as far as this impl goes, similar to paa-trust-store-path we can set the dac-revocation-set-path. We have [test implementation](https://github.com/project-chip/connectedhomeip/blob/master/src/credentials/attestation_verifier/TestDACRevocationDelegateImpl.h) of the [DeviceAttestationRevocationDelegate](https://github.com/project-chip/connectedhomeip/blob/master/src/credentials/attestation_verifier/DeviceAttestationVerifier.h#L426) which uses the path based logic.